timemachine/cpp/src/kernels/k_neighborlist.cuh

```diff
@@ -223,9 +231,11 @@ void __global__ k_find_blocks_with_ixns(
     // Determine if the row block can be translated into a periodic box
     // to optimize distance calculations
     // https://github.com/proteneer/timemachine/issues/320
-    const bool single_periodic_box =
-        (0.5f * bx - row_bb_ext_x >= cutoff && 0.5f * by - row_bb_ext_y >= cutoff &&
-         0.5f * bz - row_bb_ext_z >= cutoff);
+    // Only possible when performing upper triangle, as otherwise indices may not be ordered
+    // and will cause the kernel to hang
```

Comment:
Q: I'm not sure I understand what about having unordered indices would cause the kernel to hang -- do you understand why this happens? My expectation was that it would just run slower due to not having tight bounding boxes, similarly to what happens when hilbert sorting is disabled

Reply:
You were right, wasn't necessary. Was from a previous implementation where there was only a single set of bounding boxes, but since the bounding boxes are now separate and built on the order of the idxs it is no longer invalid.